### PR TITLE
AT_005.04_EntityFieldsOps_DeleteRecord

### PR DESCRIPTION
**Pull request by TorGash24**
US: https://trello.com/c/NFkak7w9/76-us00504-entity-fields-ops-delete
TC: https://trello.com/c/RfFQLJCx/133-tc00504-entity-fields-ops-delete
AT: https://trello.com/c/ftfjuHS3/386-at00504-entity-fields-ops-delete

### CODE REVIEW

File: src/test/java/EntityFieldsOps3Test.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            Assert.assertEquals(actualDataRecord[i], expectedDataRecord[i]);
+        }
+
+        getDriver().findElement(By.xpath("//div[@class='dropdown pull-left']")).click();
+        getWait().until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//a[text()='delete']"))).click();
+
+        getDriver().findElement(By.xpath("//i[text()='delete_outline']")).click();
+        assertPaginationInfo();
+
+        Assert.assertEquals(getDriver().findElement(By.xpath("//span[text()='Reference: ']/b")).getText(), label);
```
Comment (ArinaJur) [inline src/test/java/EntityFieldsOps3Test.java:94]: For refactoring please consider to remove extra line.

**Review by ArinaJur — APPROVED** [review]
Good Job!

File: src/test/java/EntityFieldsOps3Test.java (hunk 1 above)
Comment (ArinaJur) [inline src/test/java/EntityFieldsOps3Test.java:94]: For refactoring please consider to remove extra line.

**Review by mkament — APPROVED** [review]
Line 9 space is needless and should be removed and "actualCreateRecord" renamed into "actualRecordCreated"(later). All else is good.

**Pull request merged**